package/MDAnalysis/analysis/align.py

```
@@ -359,27 +406,152 @@ def alignto(mobile, reference, select="all", mass_weighted=False,
+            subdomain to be fitted against
+        filename : string, optional
+            Provide a filename for results to be written to
+        prefix : string, optional
+            Provide a string to prepend to filename for results to be written
+            to
+        mass_weighted : boolean, optional
+            Boolean, if true will rmsd will be mass-weighted corresponding to
+            the atoms selected in the reference trajectory
+        tol_mass : float, optional
+            Tolerance given to `get_matching_atoms` to find appropriate atoms
+        strict : boolean, optional
+            Force `get_matching_atoms` to fail if atoms can't be found using exact
+            methods
+        force : boolean, optional
+            Force overwrite of filename for rmsd-fitting
+        quiet : boolean, optional
+            Set logger to show minimal information
+        start : int, optional
+            First frame of trajectory to analyse, Default: 0
+        stop : int, optional
+            Last frame of trajectory to analyse, Default: -1
+        step : int, optional
+            Step between frames to analyse, Default: 1
+
+        """
+
+        self.mobile = mobile
+        self.reference = reference
+        frames = self.mobile.trajectory
+
+        select = rms.process_selection(select)
+        self.ref_atoms = self.reference.select_atoms(*select['reference'])
+        self.mobile_atoms = self.mobile.select_atoms(*select['mobile'])
+        self.natoms = self.mobile_atoms.n_atoms
+
+        kwargs.setdefault('remarks', 'RMS fitted trajectory to reference')
+
+        if filename is None:
+            path, fn = os.path.split(frames.filename)
+            filename = os.path.join(path, prefix + fn)
+
+        if os.path.exists(filename) and not force:
+            logger.warn("{0} already exists and will NOT be overwritten; use force=True if you want this".format(filename))
```

You should raise an exception here. Obviously the aligned trajectory will not be written. Something is seriously going wrong.